pom.xml

```diff
@@ -57,8 +57,8 @@
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.build.timestamp.format>yyyyMMddhhmm</maven.build.timestamp.format>
-    <maven.version>3.0</maven.version>
-    <bnd.version>5.3.0</bnd.version>
+    <maven.version>3.9.0</maven.version>
```

Comment:
Upgrading this breaks compatibility with Maven 3.0.x, which isn't such big problem, but do you really need it?

Reply:
not really. Do you mean just the maven.version or as well the bnd.version upgrade?

Reply:
The maven.version, but I'm thinking we should upgrade that anyway, and revert at some point if it ever breaks something someone needs...

Reply:
ok, agreed.